### DISCUSSION TITLE
YouTube fast forward/rewind behavior

### DISCUSSION THREAD

**Discussion by SameenAhnaf** [libre-tube/LibreTube#3751] [General]
### Describe your suggested feature

Current behavior for double tap gesture does not indicate the exact progress made. Double tap needs to be done to move every 10 seconds. It would have been better if every single tap within the arc could make 10 seconds. Each tap outside the arc should cancel 10 seconds progress too. Ripple effects within the arc would make the experience more immersive too.

### Other details

https://github.com/libre-tube/LibreTube/assets/79245641/a411640a-00c6-485e-a964-5b3c23f3c210

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
The title and the issue description are about different topics?
Anyways I personally prefer the current appearance of the double tap gesture and don't think any efforts to change it would be worth it.
Hence, I'll convert this to a discussion.

**Comment by ghost** [comment]
Btw ripple effect is not necessary for now. Just the behavior of forward/rewind `10t` seconds in `t+1` taps is enough. It would make user interaction much efficient.
